platform/admin/core/admin-core-impl/src/test/resources/custom.system.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Global URL Properties
+#
+# The httpsPort and httpPort are the ports that the system will run on
+org.codice.ddf.system.protocol=https://
+org.codice.ddf.system.hostname=localhost
+org.codice.ddf.system.httpsPort=1234
+org.codice.ddf.system.httpPort=5678
+org.codice.ddf.system.port=${org.codice.ddf.system.httpsPort}
+org.codice.ddf.system.rootContext=/services
+# external properties are useful when DDF is behind a proxy and
+# the URL a user sees is different than the local url DDF uses
+org.codice.ddf.external.protocol=${org.codice.ddf.system.protocol}
+org.codice.ddf.external.hostname=${org.codice.ddf.system.hostname}
+org.codice.ddf.external.httpsPort=${org.codice.ddf.system.httpsPort}
+org.codice.ddf.external.httpPort=${org.codice.ddf.system.httpPort}
+org.codice.ddf.external.port=${org.codice.ddf.external.httpsPort}
+org.codice.ddf.external.context=
+org.codice.ddf.system.siteName=site
+org.codice.ddf.system.siteContact=contact
+org.codice.ddf.system.version=version
+org.codice.ddf.system.organization=org
```

Comment:
✏️ Update this test file to only include properties that SystemPropertiesAdmin cares about
```
org.codice.ddf.system.hostname=localhost
org.codice.ddf.system.httpsPort=8993
org.codice.ddf.system.httpPort=8181
org.codice.ddf.external.hostname=${org.codice.ddf.system.hostname}
org.codice.ddf.external.httpsPort=${org.codice.ddf.system.httpsPort}
org.codice.ddf.external.httpPort=${org.codice.ddf.system.httpPort}
org.codice.ddf.system.siteName=ddf.distribution
org.codice.ddf.system.siteContact=
org.codice.ddf.system.version=2.14.0-SNAPSHOT
org.codice.ddf.system.organization=Codice Foundation
```